src/lib/Twig/Components/Icon.php

```diff
@@ -12,9 +12,7 @@
 use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
 use Symfony\UX\TwigComponent\Attribute\PostMount;
 
-#[AsTwigComponent(
-    template: '@ibexadesign/design_system/components/Icon.html.twig'
```

Comment:
https://github.com/ibexa/design-system-twig/blob/9209df7f33355fb85281c91ea450aba2230ad823/src/bundle/Templating/Twig/TemplateExtension.php#L16-L18

Are you :100: sure that this will work as intended?

There seems to be a difference in template location, unless I'm looking at the wrong place?

Reply:
For sure, it is working 
<img width="1462" height="165" alt="image" src="https://github.com/user-attachments/assets/c85bc5e7-e21c-4db6-aaab-9911a8ae668c" />
Also, other Twig components, like Label or Expander, don't have a template defined.